handsontable/src/core/hooks/constants.js

```diff
@@ -3204,6 +3204,24 @@ export const REGISTERED_HOOKS = [
    */
   'afterDialogFocus',
 
+  /**
+   * Fired by {@link Loading} plugin before showing the loading indicator. This hook is fired when {@link Options#loading}
+   * option is enabled. The callback can return `false` to prevent the loading indicator from being shown.
+   *
+   * @since 16.1.0
+   * @event Hooks#beforeLoadingShow
+   */
+  'beforeLoadingShow',
+
+  /**
+   * Fired by {@link Loading} plugin after showing the loading indicator. This hook is fired when {@link Options#loading}
+   * option is enabled.
+   *
+   * @since 16.1.0
+   * @event Hooks#afterLoadingShow
+   */
+  'afterLoadingShow',
```

Comment:
Is it ok that there is `{after/before}LoadingShow` hook but not `LoadingHide`? For the sake of consistency, I'd implement both.